### DISCUSSION TITLE
Renovate security updates for GitHub Actions using Dependabot Alerts supported?

### DISCUSSION THREAD

**Discussion by HonkingGoose** [renovatebot/renovate#17095] [Suggest an Idea]
Does Renovate already know how to read the _new_ Vulnerability Alerts for GitHub Actions? GitHub can now raise alerts when you're using a vulnerable GitHub Action runner. [^gh-blog] 

Relevant snippet from the `vulnerability Alerts` docs: [^renovate-docs]

> Renovate can read GitHub's Vulnerability Alerts to customize its Pull Requests.
> For this to work, you must enable the [Dependency graph](https://docs.github.com/en/code-security/supply-chain-security/understanding-your-software-supply-chain/about-the-dependency-graph#enabling-the-dependency-graph), and [Dependabot alerts](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-security-and-analysis-settings-for-your-repository).

I wanted to discuss this first before creating a feature request. 😉 

[^gh-blog]: [GitHub blog, Dependabot now alerts for vulnerable GitHub Actions](https://github.blog/2022-08-09-dependabot-now-alerts-for-vulnerable-github-actions/)
[^renovate-docs]: https://docs.renovatebot.com/configuration-options/#vulnerabilityalerts

**Comment by viceice** [comment]
We probably need to update those
https://github.com/renovatebot/renovate/blob/74922000140ccad4c476d0dfbe0522769ea17487/lib/workers/repository/init/vulnerability.ts#L66-L73
https://github.com/renovatebot/renovate/blob/74922000140ccad4c476d0dfbe0522769ea17487/lib/workers/repository/init/vulnerability.ts#L94-L102

**Reply by HonkingGoose** [reply]
I created an issue to track this work, see:

- #17250
